### PR TITLE
Change the name of the machine-remediation-operator repository

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

**What this PR does / why we need it**:

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pages/machine-remediation-operator.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
 title:
-permalink: /machine-remediation-operator/
+permalink: /machine-remediation/
```
Comment (iranzo) [inline pages/machine-remediation-operator.md]: If we change the permalink users coming to the old url will fail to find it, probably set up an additional file to be redirecting to this one?

Adding the Redirect header to this one?

Comment (iranzo) [reply pages/machine-remediation-operator.md]: Check quickstart_minikube for it's header:

redirect_from: "/get_kubevirt/" 

in the yaml preamble

Comment (cynepco3hahue) [reply pages/machine-remediation-operator.md]: @iranzo Renamed the pages file and added re-direct directive.

**Comment by iranzo** [comment]
Trying to access: https://deploy-preview-388--kubevirtio.netlify.com/machine-remediation-operator/

Redirects to kubevirt.io/machine-remediation/ so...

/lgtm

**Comment by iranzo** [comment]
@cynepco3hahue do you want to add something more or we can merge it?

**Comment by cynepco3hahue** [comment]
@iranzo Nope that all that I wanted to add:) And thanks for fast review.

**Pull request merged**